adafruit_mcp9600.py

```diff
@@ -82,6 +193,77 @@ def __init__(self, i2c, address=_DEFAULT_ADDRESS, tctype="K", tcfilter=0):
         self.buf[1] = tcfilter | (ttype << 4)
         with self.i2c_device as tci2c:
             tci2c.write(self.buf, end=2)
+        if self._device_id != 0x40:
+            raise RuntimeError("Failed to find MCP9600 - check wiring!")
+
+    def alert_config(self, *, alert_number, alert_temp_source, alert_temp_limit, alert_hysteresis,
+                     alert_temp_direction, alert_mode, alert_state):
+        """Configure a specified alert pin. Alert is enabled by default when alert is configured.
+        To disable an alert pin, use ``alert_disable``.
+
+        :param int alert_number: The alert pin number. Must be 1-4.
+        :param alert_temp_source: The temperature source to monitor for the alert. Options are:
+                                  ``THERMOCOUPLE`` or ``AMBIENT``.
+        :param float alert_temp_limit: The temperature in degrees Celsius at which the alert should
+                                       trigger. For rising temperatures, the alert will trigger when
+                                       the temperature rises above this limit. For falling
+                                       temperatures, the alert will trigger when the temperature
+                                       falls below this limit.
+        :param float alert_hysteresis: The alert hysteresis range. Must be 0-255 degrees Celsius.
+                                       For rising temperatures, the hysteresis is below alert limit.
+                                       For falling temperatures, the hysteresis is above alert
+                                       limit. See data-sheet for further information.
+        :param alert_temp_direction: The direction the temperature must change to trigger the alert.
+                                     Options are ``RISING`` (heating up) or ``FALLING`` (cooling
+                                     down).
+        :param alert_mode: The alert mode. Options are ``COMPARATOR`` or ``INTERRUPT``. If setting
+                           mode to ``INTERRUPT``, use ``alert_interrupt_clear`` to clear the
+                           interrupt flag.
```

Comment:
In comparator mode, the pin will follow the alert, so if the temperature drops, for example, the alert pin will go back low. In interrupt mode, by comparison, once the alert goes off, you must manually clear it.